src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class FileWork {
+    private static final String START_LETTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> listOfWords = new ArrayList<>();
+
+        File file = new File(fileName);
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            String value = reader.readLine();
+            while (value != null) {
+                for (String word : value.split(" ")) {
```

Comment:
let's find a regex that will split in such a way that we won't need to do `.replaceAll("[!?.,]", "")`